README.md

```diff
@@ -241,7 +229,8 @@ Here's a reference of which SwiftLint version to use for a given Swift version.
 | Swift 3.x       | SwiftLint 0.25.1                 |
 | Swift 4.0-4.1.x | SwiftLint 0.28.2                 |
 | Swift 4.2.x     | SwiftLint 0.35.0                 |
-| Swift 5.x       | Latest                           |
+| Swift 5.0-5.2.x | SwiftLint 0.43.0                 |
```

Comment:
The tag history of https://hub.docker.com/r/norionomura/swiftlint looks like the following.
https://hub.docker.com/r/norionomura/swiftlint/tags?name=swift-5.0
https://hub.docker.com/r/norionomura/swiftlint/tags?name=swift-5.1
```suggestion
| Swift 5.0.x     | SwiftLint 0.39.2                 |
| Swift 5.1.x     | SwiftLint 0.41.0                 |
| Swift 5.2.x     | SwiftLint 0.43.0                 |
```

Reply:
Thanks for flagging! I wonder if this table is actually useful to most people though. Maybe I should just remove it?

Reply:
I just removed it.

These weren't exactly accurate because SwiftLint should still work with older Swift versions long after we update the minimum version of Swift needed to _build_ the tool. So I doubt this table was very helpful to anyone.